src/helpers/get-comments.ts

```diff
@@ -60,12 +64,16 @@ export const getOthersComments = ({
     return comments
 }
 
-export const getLongComments = ({ data }: { data: any }): Comment[] => {
-    const rawComments = data?.issueComments?.nodes.filter(
-        (comment: any) => comment.body.length > 500
-    )
+export const getLongComments = ({
+    data
+}: {
+    data: IssueCommentNodes[]
+}): Comment[] => {
+    data = data !== undefined ? data : []
+
+    const rawComments = data?.filter((comment) => comment.body.length > 500)
```

Comment:
no hardcoded value. 500 should be a constant in config
```suggestion
    const rawComments = data?.filter((comment) => comment.body.length > MAX_COMMENT_LENGTH
```